src/bin/miri.rs

```diff
@@ -44,22 +44,25 @@ impl<'a> CompilerCalls<'a> for MiriCompilerCalls {
             fn extract_str(lit: &syntax::ast::Lit) -> syntax::parse::token::InternedString {
                 match lit.node {
                     syntax::ast::LitKind::Str(ref s, _) => s.clone(),
-                    _ => panic!("attribute values need to be strings"),
+                    _ => bug!("attribute values need to be strings"),
```

Comment:
This should actually be a span_err, because it is a miri user fault, not miri progrmmer fault.